### ISSUE TITLE
Prepare deployable application skeleton (app + db)

### ISSUE THREAD

**Issue by pnpavlov** [gardenlinux/glvd#79]
Package all required components of the GLVD backend in a deployment manifest that can be applied to a Kubernetes environment. That includes but is not limited to:
- Helm charts for the GLVD API and for the database deployed in same namespace.
- The components of the chart need to decoupled in a way that enables managing them separately and exposing them within the environment over services. For now we can assume it's single namespace.
 
Not in scope:
- No High Availability or Disaster Recovery requirements to be covered during this early development stage.

**Comment by pnpavlov** [comment]
@fwilhe , can you please confirm this is done and can be closed? Do we have a deployment manifest with or without helm that could be referred to? If so, please refer and close?

**Comment by fwilhe** [comment]
Yes we have a working k8s deployment [here](https://github.com/gardenlinux/glvd/tree/main/deployment/k8s). Did not convert this to helm so far because I don't think it would bring us a huge benefit, but @MrBatschner already sketched out what this would look like. It is a question of priority, and for me this has close to zero priority currently because the existing solution works well enough for now.
